packages/cw-derive/src/check_generics.rs

```diff
@@ -17,12 +17,24 @@ impl<'g> CheckGenerics<'g> {
     pub fn used(self) -> Vec<&'g Ident> {
         self.used
     }
+
+    /// Returns split between used and unused generics
+    pub fn used_unused(self) -> (Vec<&'g Ident>, Vec<&'g Ident>) {
```

Comment:
Can you rename to `split_used_unused`?
The name is confusing until reading comment and code.